src/inputs/buttons/toggle-button/PToggleButton.vue

```diff
@@ -1,82 +1,93 @@
 <template>
-    <span class="toggle-button"
-          :class="{'disabled': disabled}"
+    <label class="p-toggle-button"
+           :class="{'disabled': props.disabled}"
     >
-        <toggle-button
-            :value="value"
-            :sync="sync"
-            :color="colors"
-            :width="32"
-            :height="16"
-            :margin="2"
-            :disabled="disabled"
-            v-on="$listeners"
-        />
-    </span>
+        <input role="switch"
+               type="checkbox"
+               class="slider"
+               :class="[props.styleType]"
+               :disabled="props.disabled"
+               :checked="props.value"
+               @change="handleChangeToggle"
+        >
+        <span v-if="props.label !== '' && props.label !== undefined"
```

Comment:
nitpick: I think this code can be shorten like below:

```suggestion
        <span v-if="!!props.label"
```